x-pack/legacy/plugins/apm/typings/common.ts

```diff
@@ -4,6 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import '../../infra/types/rison_node';
+import {} from '../../reporting/public/components/report_listing';
```

Comment:
What do we import from reporting that needs types?

Reply:
`EuiBasicTable`. I'll add a comment to clarify.